src/targets/gpu/convolution.cpp

```diff
@@ -121,6 +152,69 @@ shape miopen_convolution::find(context& ctx, const shape& output_shape, std::vec
         MIGRAPHX_THROW("MIOpen Convolution: find convolution failed");
     algo = perf.fwd_algo;
 
+    if(enabled(MIGRAPHX_MIOPEN_FIND_2{}))
+    {
+        miopenProblem_t conv_problem;
+        status = miopenCreateConvProblem(&conv_problem, cd.get(), miopenProblemDirectionForward);
```

Comment:
This should use a manage pointer and use `make_obj` to construct the object.

Reply:
Done.